### ISSUE TITLE
Questions about the training details

### ISSUE THREAD

**Issue by yqjiang6** [Yang-Liu1082/InvDN#24]
Hi! This is a really interesting work in image denoising, and I try to reproduce the results using your code. However, I have some questions about the details of your code:

1. The train_InvDN.yaml file is different from the description in the paper, e.g., batch size of 28 in yaml vs 14 in the paper, learning rate steps 100k in yaml vs 50k in paper. Which configuration should I use to reproduce your result in the paper? or  Could you please provide the original yaml file of your paper?
2. About the loss function: the backward loss still contains the gradient loss and the SSIM loss. Should I comment them or not?
3. Just to make sure how to prepare the SSID training data. Is it right to crop each training image into 100 patches with the patch size of 512 using Crop_SIDD.py?

Thanks for your patience to read my questions, and I'm looking forward to receiving your reply!
